src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlConstants.cs

```diff
@@ -19,6 +19,8 @@ internal static class XamlConstants
 		public const string BaseXamlNamespace = "Windows.UI.Xaml";
 		public const string UnoXamlNamespace = "Windows.UI.Xaml";
 
+		public const int MaxFluentResourcesVersion = 2;
+
```

Comment:
Maybe this should be define somewhere _closer_ to the fluent projects?

Reply:
We'll change it once v3 is available. The problem here is the availability of this variable from two scopes (inside the generators and in the fluent assemblies), pulling the value is more trouble than it's worth for now.